src/storage/src/hummock/compactor/mod.rs

```diff
@@ -677,9 +678,24 @@ pub fn start_shared_compactor(
                                         .report_compaction_task(report_compaction_task_request)
                                         .await
                                     {
-                                        Ok(_) => {}
+                                        Ok(_) => {
+                                            // TODO: remove this method after we have running risingwave cluster with fast compact algorithm stably for a long time.
+
+                                            if context.storage_opts.check_fast_compaction_result
+                                                && !compact_task.sorted_output_ssts.is_empty()
+                                                && compact_task.task_status() == TaskStatus::Success
+                                                && let Err(e) = check_compaction_result(&compact_task, context.clone()).await
+                                            {
+                                                tracing::error!(
+                "Failed to check fast compaction task {} because: {:?}",
```

Comment:
Does typo log？

we should enable the `compaction_check` for both fast and default

Reply:
Fixed